src/haywire/http_connection.h

```diff
@@ -1,18 +1,2 @@
 #pragma once
-#include "uv.h"
-#include "http_parser.h"
-#include "http_request.h"
-
-typedef struct
-{
-    uv_tcp_t stream;
-    http_parser parser;
-    uv_write_t write_req;
-    http_request* request;
-    char current_header_key[1024];
-    int current_header_key_length;
-    char current_header_value[1024];
-    int current_header_value_length;
-    int keep_alive;
-    int last_was_value;
-} http_connection;
+#include "http.h"
```

Comment:
Don't you want to remove this file altogether? I'm assuming that since it's not under `include` that it's not intended to be used by a client of the library.